src/nspawn/nspawn-mount.c

```diff
@@ -342,11 +349,17 @@ int mount_all(const char *dest,
         for (k = 0; k < ELEMENTSOF(mount_table); k++) {
                 _cleanup_free_ char *where = NULL, *options = NULL;
                 const char *o;
+                bool fatal = (mount_table[k].mount_settings & MOUNT_FATAL) > 0;
+                bool in_userns = (mount_table[k].mount_settings & MOUNT_IN_USERNS) > 0;
+                bool use_userns = (mount_table[k].mount_settings & MOUNT_USE_USERNS) > 0;
+
+                if (((mount_settings ^ mount_table[k].mount_settings) & MOUNT_IN_USERNS) > 0)
```

Comment:
why the `> 0` check here? you can just drop that and rely on C's downgrade-to-bool feature here. After all, this is actually a bool-like expression that you are using here, hence using it as a bool if you can makes a ton of sense.